erpnext/public/js/utils/barcode_scanner.js

```diff
@@ -34,6 +36,8 @@ erpnext.utils.BarcodeScanner = class BarcodeScanner {
 		//     batch_no: "LOT12", // present if batch was scanned
 		//     serial_no: "987XYZ", // present if serial no was scanned
 		//     uom: "Kg", // present if barcode UOM is different from default
+		//     warehouse: "Store-001", // present if warehouse was scanned (location-first scanning)
+		//     is_warehouse: true, // present if a warehouse was scanned
```

Comment:
The difference between these two is not obvious from the description. I assume `warehouse` is available if any warehouse was scanned before and `is_warehouse` means that this particular scan was a warehouse?